src/Configuration.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * Common configurations for Proxy PagoPa and external resources
+ */
+
+import * as t from "io-ts";
+import { WithinRangeNumber } from "italia-ts-commons/lib/numbers";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+
+const localhost = "http://localhost";
+
+export const CONFIG = {
+  // RESTful Webservice configuration
+  CONTROLLER: {
+    PORT: process.env.PAGOPAPROXY_PORT || 3000,
+    HOST: process.env.PAGOPAPROXY_HOST || localhost,
+    ROUTES: {
+      RESTFUL: {
+        PAYMENTS_CHECK: "/payment/check",
+        PAYMENTS_ACTIVATION: "/payment/activation"
+      },
+      SOAP: {
+        PAYMENTS_STATUS_UPDATE: "cdInfoWisp"
+      }
+    }
+  },
+
+  // PagoPa Configuration
+  PAGOPA: {
+    HOST: process.env.PAGOPA_HOST || localhost,
+    PORT: process.env.PAGOPA_PORT || 3001,
+    SERVICES: {
+      PAYMENTS_CHECK: "nodoVerificaRPT",
+      PAYMENTS_ACTIVATION: "nodoAttivaRPT"
+    },
+    IDENTIFIER: {
+      IDENTIFICATIVO_PSP: "AGID_01",
```

Comment:
where do this values come from ? probably it's better to define some const(s), add a comment for each const that explain the reason for the value, then use them here